pkg/network/config/config.go

```diff
@@ -372,6 +385,8 @@ func New() *Config {
 		log.Info("network tracer DNS inspection disabled by configuration")
 	}
 
+	c.ServiceMonitoringEnabled = c.ServiceMonitoringEnabled || c.DataStreamsEnabled
```

Comment:
If only DSM is enabled, this will force HTTP+HTTPS monitoring on. Is that intentional?

Reply:
At the moment yes.